frontend/src/components/pages/CampOverview/index.tsx

```diff
@@ -70,6 +89,95 @@ const CampOverviewPage = (): JSX.Element => {
 
   const openManageSessionsModal = () => onOpen();
 
+  const getUpdateCampSessionsRequestArray = (
+    updatedCapacities: Map<string, string>,
+  ): Array<UpdateCampSessionsRequest> => {
+    const requests: Array<UpdateCampSessionsRequest> = [];
+
+    updatedCapacities.forEach((capacity, id) => {
+      const updatedCapacityExists = capacity.trim().length > 0;
```

Comment:
not too bothered but why not use a map function to build the array of Requests? honestly dealing with the Map<string, string> is a pain so if it's causing too much issues this works, otherwise, I can see a use of filter + map to build the array

Reply:
Ehh not sure, I'll merge for now